nav2_planner/src/planner_server.cpp

```diff
@@ -481,12 +481,73 @@ PlannerServer::computePlan()
     }
 
     action_server_pose_->succeeded_current(result);
+  } catch (nav2_core::StartOccupied & ex) {
+    RCLCPP_WARN(
+      get_logger(), "%s plugin failed to plan path. "
+      "Start Pose (%.2f, %.2f) was occupied: \"%s\"",
+      goal->planner_id.c_str(),
+      start.pose.position.x,
+      start.pose.position.y,
+      ex.what());
+    result->error_code = nav2_msgs::action::ComputePathToPose::Goal::START_OCCUPIED;
+    action_server_pose_->terminate_current(result);
+  } catch (nav2_core::GoalOccupied & ex) {
+    RCLCPP_WARN(
+      get_logger(), "%s plugin failed to plan path. Goal Pose (%.2f, %.2f) was occupied: \"%s\"",
+      goal->planner_id.c_str(),
+      goal->goal.pose.position.x,
+      goal->goal.pose.position.y,
+      ex.what());
+    result->error_code = nav2_msgs::action::ComputePathToPose::Goal::GOAL_OCCUPIED;
+    action_server_pose_->terminate_current(result);
+  } catch (nav2_core::NoValidPathCouldBeFound & ex) {
+    RCLCPP_WARN(
+      get_logger(), "%s plugin failed to plan path. No Valid Path found: \"%s\"",
```

Comment:
Add in the goal pose, it helps understand which goal request this is in reference to